chapters/6-analysis.tex

```diff
@@ -106,84 +105,82 @@ \subsection{Fitting construction into the data workflow}
 	operations are commonly called ``merges'' in Stata, and
-	``joins'' in R's \texttt{tidyverse} dialect. 
+	``joins'' in R's \texttt{tidyverse} dialect.
 	We will use the first term on the next few paragraphs.}
-To understand how to perform such operations, 
+To understand how to perform such operations,
 you will need to consider the unit of observation for each dataset,
 and their respective identifying variables.
 Merges are common sources of missing values,
 and can also result in observations being dropped.
 To avoid unintentional changes to your data,
 pay close attention to merge results,
-and add checks to your code verifying that they match what you expect.
+and add checks to your code to very that they match what you expect.
 
-% Merging data sets with the same unit of observation 
+% Merging data sets with the same unit of observation
 If the datasets you need to merge have the same unit of observation,
 merging may be straightforward.
 The simplest case is when the datasets have the same unit of observation
-and use a consistent, uniquely and fully identifying ID variable. 
+and use a consistent, uniquely and fully identifying ID variable.
 For example, in the case of administrative data for firms,
 you may merge data from different years using the firm taxpayer code
-(or, preferably, a deidentified version of it).
+(or, preferably, a de-identified version of it).
 In such cases, the main source of unexpected results are unmatched observations.
 Fortunately, it is straightforward to avoid mistakes in this setting:
 before writing code to combine the datasets,
 write pseudocode to understand which observations you expect to be
 matched or not, and why.
-When possible, determine exactly which and how many 
+When possible, determine exactly which and how many
 matched and unmatched observations should result from the merge.
-For example, first that were created in the second year should not be
-present in the first year's data, 
-and firms that closed during the first year should not be observed in the second year's data. 
+For example, firms that were created in the second year should not be
+present in the first year's data,
+and firms that closed during the first year should not be observed in the second year's data.
 Then think carefully about whether you want to keep matched and unmatched
-observations, or only specific matching outcomes,
- for example, to create a balanced panel,
-and add that to your pseudocode as well.
-Now you can merge the datasets, 
+observations, or only specific matching outcomes (e.g. to create a balanced panel).
+Add that to your pseudocode as well.
+Now you can merge the datasets,
 and test that the outcome matches your expectations.
-Add comments to explain any exceptions, 
+Add comments to explain any exceptions,
 and make it so the code will return an error in case unexpected results show up in future runs.
 This approach can be extended to other types of merges between two related datasets.
```

Comment:
A lot of this is discussed in chapter 3 in relation to data flow charts.

Reply:
Done in 06f9055